src/main/kotlin/org/rust/ide/inspections/RsNamingInspection.kt

```diff
@@ -41,6 +39,8 @@ abstract class RsNamingInspection(
             *fixes)
     }
 
+    open fun isLintSuppressedFor(element: PsiElement): Boolean = false
```

Comment:
Note, there is `com.intellij.codeInspection.InspectionProfileEntry#isSuppressedFor(com.intellij.psi.PsiElement)`
I think we want to implement this meatho instead of introducing new API

Reply:
Oops, thanks.